dask/array/stats.py

```diff
@@ -0,0 +1,519 @@
+    if nan_policy != 'propagate':
+        raise NotImplementedError("`nan_policy` other than 'propagate' "
+                                  "have not been implemented.")
+
+    n = a.shape[axis]
+    df = float(n - 1)
+
+    d = (a - b).astype(np.float64)
+    v = da.var(d, axis, ddof=1)
+    dm = da.mean(d, axis)
+    denom = da.sqrt(v / float(n))
+
+    with np.errstate(divide='ignore', invalid='ignore'):
+        t = da.divide(dm, denom)
+    t, prob = _ttest_finish(df, t)
+
+    return delayed(Ttest_relResult, nout=2)(t, prob)
+
+
+@doc_wraps(scipy.stats.chisquare)
+def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
+    return power_divergence(f_obs, f_exp=f_exp, ddof=ddof, axis=axis,
+                            lambda_="pearson")
+
+
+@doc_wraps(scipy.stats.power_divergence)
+def power_divergence(f_obs, f_exp=None, ddof=0, axis=0, lambda_=None):
+
+    if isinstance(lambda_, str):
+        # TODO: public api
+        if lambda_ not in scipy.stats.stats._power_div_lambda_names:
+            names = repr(list(scipy.stats.stats._power_div_lambda_names.keys()))[1:-1]
+            raise ValueError("invalid string for lambda_: {0!r}.  Valid strings "
+                             "are {1}".format(lambda_, names))
+        lambda_ = scipy.stats.stats._power_div_lambda_names[lambda_]
+    elif lambda_ is None:
+        lambda_ = 1
+
+    if f_exp is not None:
+        # f_exp = np.atleast_1d(np.asanyarray(f_exp))
+        pass
+    else:
+        f_exp = f_obs.mean(axis=axis, keepdims=True)
+
+    # `terms` is the array of terms that are summed along `axis` to create
+    # the test statistic.  We use some specialized code for a few special
+    # cases of lambda_.
+    if lambda_ == 1:
+        # Pearson's chi-squared statistic
+        terms = (f_obs - f_exp)**2 / f_exp
+    elif lambda_ == 0:
+        # Log-likelihood ratio (i.e. G-test)
+        terms = 2.0 * _xlogy(f_obs, f_obs / f_exp)
+    elif lambda_ == -1:
+        # Modified log-likelihood ratio
+        terms = 2.0 * _xlogy(f_exp, f_exp / f_obs)
+    else:
+        # General Cressie-Read power divergence.
+        terms = f_obs * ((f_obs / f_exp)**lambda_ - 1)
+        terms /= 0.5 * lambda_ * (lambda_ + 1)
+
+    stat = terms.sum(axis=axis)
+
+    num_obs = _count(terms, axis=axis)
+    # ddof = asarray(ddof)
+    p = delayed(distributions.chi2.sf)(stat, num_obs - 1 - ddof)
+
+    return delayed(Power_divergenceResult, nout=2)(stat, p)
+
+
+@doc_wraps(scipy.stats.skew)
+def skew(a, axis=0, bias=True, nan_policy='propagate'):
+    if nan_policy != 'propagate':
+        raise NotImplementedError("`nan_policy` other than 'propagate' "
+                                  "have not been implemented.")
+
+    n = a.shape[axis]
+    m2 = moment(a, 2, axis)
+    m3 = moment(a, 3, axis)
+    zero = (m2 == 0)
+    vals = da.where(~zero, m3 / m2**1.5, 0.)
+    # vals = da.where(~zero, (m2, m3),
+    #                 lambda m2, m3: m3 / m2**1.5,
+    #                 0.)
+    if not bias:
```

Comment:
I've changed tests so that they cover this code (and below).  This raises two errors in the following line:

```python
da.tile(nval, n_replace)[:n_replace]
```

1. da.tile needs an integer, not a tuple of integers (we maybe need to extend da.tile here?)
2.  After indexing with a dask.array we lose shape information, and so can no longer slice